packages/components/scroll-area/src/props.ts

```diff
@@ -298,7 +299,7 @@ export type ScrollAreaScrollbarImplEmits = {
 }
 
 /* -------------------------------------------------------------------------------------------------
- *  ScrollAreaScrollbarAxisX - scroll-area-scrollbar-axis-x.ts
+ *  ScrollAreaScrollbarAxis
```

Comment:
`scroll-area-scrollbar-axis-x.ts` please write file path

Reply:
Actually the code below this comment is used by `ScrollAreaScrollbarAxisX` and `ScrollAreaScrollbarAxisY`.

Do you want it to be like this?

`ScrollAreaScrollbarAxisX/Y - scroll-area-scrollbar-axis-x/y.ts`

Reply:
yes please